test-case-reporting/index.ts

```diff
@@ -35,9 +38,14 @@ app.get('/test-results/history/:testCaseId', (req, res) => {
   res.send(`Test history for test with name/ID ${testCaseId}`);
 });
 
-app.post('/report', function (req, res) {
-  const report: KarmaReporter.TestResultReport = req.body;
-  res.send(`Travis report! The request body is below.\n\n${report}`);
+app.post('/report', async (req, res) => {
```

Comment:
Lets keep these changes to the server/api handler separate from the actual interface to the DB

Reply:
SGTM; should I rebase to remove that commit?

Reply:
We're at 27 commits now so I'd say the commit history isn't exactly pristine, you could just remove it *shrug*

Reply:
Done.